teleduck/src/teleduck/server.py

```
@@ -99,8 +101,31 @@ def handle_query(self, sql, callback=callable, **kwargs):
         self.executor.submit(self._handle_query, sql, callback, **kwargs)
 
     def handle_auth(self, user, password, host, database=None, callback=callable):
```

Pylint (3.3.7)

[refactor] Too many arguments (6/5)

(R0913)